### ISSUE TITLE
Focus indicator should be on top of features

### ISSUE THREAD

**Issue by prushforth** [Maps4HTML/Web-Map-Custom-Element#940]
The focused feature should have a focus indicator that is drawn on top of all features, but it goes over and under the other features in the rendering stack adjacent to the feature.    Should be fixable??

![image](https://github.com/Maps4HTML/Web-Map-Custom-Element/assets/2437285/eac8eb33-2271-4521-a667-0e05ea912b60)
